src/sider/frisktilarbeid/FattVedtakSkjema.tsx

```diff
@@ -50,7 +53,7 @@ export const FattVedtakSkjema = () => {
   } = methods;
 
   const fraDato = watch("fraDato");
-  const tilDato = fraDato ? addWeeks(fraDato, 12) : undefined;
+  const tilDato = addWeeks(fraDato, 12);
```

Comment:
Er det behov for å ta hensyn til bevegelige helligdager her?

Reply:
Hm, det kan hende. Vi har allerede en lapp på å ta hensyn til max-dato så kan legge det inn der.